dockclean/containers.py

```diff
@@ -19,20 +19,23 @@ def stop_all_running_containers(logger):
 
 
 def remove_all_stopped_containers(logger):
-    print("Removing all stopped containers.")
+    logger.info("Removing all stopped containers.")
+
     client = docker.from_env()
-    containers = client.containers.list(filters={"status": "exited"})
-    count = 0
-    for container in containers:
-        try:
-            container.remove()
-            logger.info(f"Removed container -> {container.name}")
-            count += 1
-        except:
-            logger.error(
-                f"Exception occurred while removing container -> {container.name}"
-            )
-    print(f"Total {count} stopped containers removed.")
+    deleted_containers = client.containers.prune()
+
+    containers = deleted_containers['ContainersDeleted']
+
+    if containers is not None:
+        [logger.info(f"Removed container -> {container}") for container in containers]
+        logger.info(f"Total {len(containers)} stopped containers removed.")
```

Comment:
And move this to the end of the function like we had before and replace `len(containers) with `count`.